rocky/rocky/views/upload_raw.py

```diff
@@ -59,9 +95,13 @@ def add_success_notification(self, success_message):
     def process_raw(self, form):
         raw_file = form.cleaned_data["raw_file"]
         mime_types = form.cleaned_data["mime_types"]
+        ooi_id = form.cleaned_data["ooi_id"]
+        input_ooi = self.octopoes_api_connector.get(ooi_id, valid_time=datetime.now(timezone.utc))
```

Comment:
This needs a try/expect to see if the input_ooi exists at the given time, if not, we should raise an error telling the user as much.

Reply:
Done, added it to the form!